arch/arm64/src/qemu/qemu_boot.c

```diff
@@ -179,7 +179,9 @@ void arm64_chip_boot(void)
    * configuration of board specific resources such as GPIOs, LEDs, etc.
    */
 
```

Comment:
Please update the comment to explain better about this #ifdef included, it will help people reading it in the future

Reply:
Hi @acassis , line 178-180 already have comments

Reply:
Yes, I saw that comment, but it still confuse for a person reading it in the future:

```
/* Perform board-specific device initialization. This would include
   * configuration of board specific resources such as GPIOs, LEDs, etc.
   */
```

Why only if CONFIG_ARCH_BOARD_QEMU_ARMV8A is defined? I only know it because you explained in the Testing here.

Maybe using:

```
#ifndef CONFIG_ARCH_BOARD_CUSTOM 
```

Could be easier to follow, what do you think?